postgres/datadog_checks/postgres/version_utils.py

```diff
@@ -0,0 +1,69 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import re
+
+import semver
+
+V8_3 = semver.parse("8.3.0")
+V9 = semver.parse("9.0.0")
+V9_1 = semver.parse("9.1.0")
+V9_2 = semver.parse("9.2.0")
+V9_4 = semver.parse("9.4.0")
+V9_6 = semver.parse("9.6.0")
+V10 = semver.parse("10.0.0")
+
+
+def get_version(db):
+    cursor = db.cursor()
+    cursor.execute('SHOW SERVER_VERSION;')
+    raw_version = cursor.fetchone()[0]
+    try:
+        version_parts = raw_version.split(' ')[0].split('.')
+        version = [int(part) for part in version_parts]
+        while len(version) < 3:
+            version.append(0)
+        return semver.VersionInfo(*version)
+    except Exception:
+        # Postgres might be in development, with format \d+[beta|rc]\d+
+        match = re.match(r'(\d+)([a-zA-Z]+)(\d+)', raw_version)
```

Comment:
is `X.YbetaZ` possible?

Reply:
I don't know. PG documentation only refers to actual releases, which are on X.Y.Z format https://www.postgresql.org/support/versioning/